cmd/bootstrapper/cmd.go

```diff
@@ -0,0 +1,127 @@
+package bootstrapper
+
+import (
+	"github.com/Dynatrace/dynatrace-bootstrapper/cmd"
+	"github.com/Dynatrace/dynatrace-bootstrapper/cmd/configure"
+	"github.com/Dynatrace/dynatrace-operator/cmd/bootstrapper/download"
+	"github.com/Dynatrace/dynatrace-operator/pkg/arch"
+	dtclient "github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
+	"github.com/Dynatrace/dynatrace-operator/pkg/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi/metadata"
+	"github.com/Dynatrace/dynatrace-operator/pkg/injection/codemodule/installer/url"
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	use = "bootstrap"
+
+	TargetFolderFlag   = cmd.TargetFolderFlag
+	TargetVersionFlag  = "version"
+	SuppressErrorsFlag = cmd.SuppressErrorsFlag
+	TechnologiesFlag   = "technologies"
+	FlavorFlag         = "flavor"
+)
+
+var (
+	targetFolder        string
+	targetVersion       string
+	areErrorsSuppressed bool
+	technologies        []string
+	flavor              string
+
+	log = logd.Get().WithName("bootstrap")
+)
+
+func New() *cobra.Command {
+	fs := afero.NewOsFs()
+
+	return newCmd(fs)
+}
+
+func newCmd(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                use,
+		RunE:               run(afero.Afero{Fs: fs}),
+		FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
+		SilenceUsage:       true,
+	}
+
+	AddFlags(cmd)
+
+	return cmd
+}
+
+func AddFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&targetFolder, TargetFolderFlag, "", "Base path where to copy the codemodule to.")
+	_ = cmd.MarkPersistentFlagRequired(TargetFolderFlag)
```

Comment:
not necessary to make it required

in case of "metadata enrichment only" you don't have `targetFolder` 
- (your PR is not capable of this scenario, which is fine, once the bootstrapper does a release, we can do it)